assignments/arrays.js

```diff
@@ -92,17 +92,32 @@ console.log(carModels);
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
 let carYears = [];
-console.log();
+for(let i=0; i<inventory.length; i++) {
+  if(inventory[i].car_year > 0){
+    carYears.push(inventory[i].car_year);
+  }
+}
+console.log(carYears);
 
 // ==== Challenge 5 ====
 // The car lot manager needs to find out how many cars are older than the year 2000. Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
 let oldCars =[];
-console.log(); 
+for(let i=0; i<inventory.length; i++) {
+  if(inventory[i].car_year <= 2000){
```

Comment:
Very close! But if we only want cars _older than_ 2000, we'd just use this:

```suggestion
  if(inventory[i].car_year < 2000){
```